app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
```

Comment:
- [ ] [OPTIONAL] good job for your styles 👏 I would also suggest that you position the remember me button on the left side because It is a convention to position them there 😃 👍 
![image](https://user-images.githubusercontent.com/90222110/206562586-ee652ac8-f3d2-4421-9f1b-b9af79f29a83.png)

Reply:
Fixed accordingly